### ISSUE TITLE
Poor screen reader compatibility

### ISSUE THREAD

**Issue by LiamOSullivan** [LiamOSullivan/bcd-dd-v2#195]
The whole site should be accessible to screen readers. This should be assessed using  an industry standard benchmarking tool
